dungeon/src/core/level/TileLevel.java

```diff
@@ -55,8 +55,6 @@ public class TileLevel implements ILevel {
   public TileLevel(Tile[][] layout) {
     this.layout = layout;
     putTilesInLists();
-    if (startTile == null) randomStart();
-    if (exitTiles.size() == 0) randomEnd();
```

Comment:
Da bin ich mir unsicher, im Grunde legen wir jetzt die verantwortung in den Level-Designer. 
Wenn wir aber von zufälligen Leveln (erstmal) weg sind, dann sollten wir m.M.n aber auch nicht zufällig Start und Endpunkte verteilen